.generate/gen-all.sh

```diff
@@ -30,14 +30,57 @@ else
   reset=''
 fi
 
+#===================================================================
+# FUNCTION trap_add ()
+#
+# Purpose:  prepends a command to a trap
+#
+# - 1st arg:  code to add
+# - remaining args:  names of traps to modify
+#
+# Example:  trap_add 'echo "in trap DEBUG"' DEBUG
+#
+# See: http://stackoverflow.com/questions/3338030/multiple-bash-traps-for-the-same-signal
+#===================================================================
+
+function trap_add() {
+    trap_add_cmd=$1; shift || fatal "${FUNCNAME} usage error"
+    new_cmd=
+    for trap_add_name in "$@"; do
+        # Grab the currently defined trap commands for this trap
+        existing_cmd=`trap -p "${trap_add_name}" |  awk -F"'" '{print $2}'`        # Define default command
+        [ -z "${existing_cmd}" ] && existing_cmd="echo exiting @ `date`"        # Generate the new command
+        new_cmd="${trap_add_cmd};${existing_cmd}"        # Assign the test
+         trap   "${new_cmd}" "${trap_add_name}" || \
+                fatal "unable to add to trap ${trap_add_name}"
+    done
+}
 function header_text {
   echo "$header$*$reset"
 }
-
-#  NOTE: The script will use the bin installed locally
-#  Ensure that you have the version required with `operator-sdk version`
+# edit the release version before execute the script
+RELEASE_VERSION=v1.1.0
+function install_bin() {
+  header_text "installing the SDK version ${RELEASE_VERSION}"
+  TMPDIR="$(mktemp -d)"
+  trap_add 'rm -rf $TMPDIR' EXIT
+  pushd "$TMPDIR"
+  git clone https://github.com/operator-framework/operator-sdk
+  cd operator-sdk
+  git checkout ${RELEASE_VERSION}
+  git checkout -b generate-release-${RELEASE_VERSION}
+  make tidy
+  make install
+  operator-sdk version
+}
```

Comment:
Seems weird that we're building it locally instead of just downloading the appropriate binary. After talking to Ish, there were some difficulty getting this download to work on both Linux and Mac consistently. For now I will let this go since it was at least attempted.